docs/src/examples/components/Tooltip/States/TooltipOpenControlledExample.shorthand.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+import { Button, Tooltip } from '@stardust-ui/react'
+import { useBooleanKnob } from '@stardust-ui/docs-components'
+
+const TooltipOpenExample = () => {
+  const [open, setOpen] = useBooleanKnob({ name: 'open-c', initialValue: true })
```

Comment:
```suggestion
  const [open, setOpen] = useBooleanKnob({ name: 'open', initialValue: true })
```

After #1489 will be merged